fastlane/Fastfile

```diff
@@ -66,14 +67,58 @@ lane :release do |options|
 
   changes = touch_changelog(release_version: version_number)
 
+  # Update SPM Repo
+  sh("git clone git@github.com:GetStream/stream-chat-swift-spm.git ../StreamSPM")
```

Comment:
Why is this done in the release and not in the publish release? It makes more sense there, that way you only edit the Package.swift once

Reply:
Yeah, good shout 👍🏼